app/src/main/java/com/plan2meet/dto/EventDTO.kt

```diff
@@ -1,11 +1,12 @@
 package com.plan2meet.dto
 
 import android.os.Build
+import android.os.Build.VERSION_CODES
 import androidx.annotation.RequiresApi
 import java.time.LocalDateTime
 import java.time.LocalDateTime.now
 
-@RequiresApi(Build.VERSION_CODES.26)
+@RequiresApi(VERSION_CODES.26)
 class EventDTO (
```

Comment:
I saw an error that said the .now() function require API level 26 , So i added the annotation. It fixed the error